lib/Data/Content/CreateContentDraftData.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Data\Content;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class CreateContentDraftData extends ValueObject
+{
+    public $contentId;
```

Comment:
In retrospect, this isn't very value objectish. In our ValueObjects, properties are meant to be protected, and set using the constructor (read only VO).

Could you update the properties for that, and adapt the action ?